src/components/Banner/Can82Banner.tsx

```diff
@@ -0,0 +1,31 @@
+import React from 'react';
+import {
+  CondensedBanner,
+  CondensedBannerWrapperHomepage,
+  CondensedBannerWrapperLocationPage,
+} from './Banner.style';
+
+//TODO(8.2) - update with correct learn more URL
+
+const Can82Banner: React.FC = () => (
+  <CondensedBanner role="banner">
+    We've made some changes to better align with the CDC.{' '}
+    <a href="/" target="_blank" rel="noopener noreferrer">
```

Comment:
Since this will be an internal link (I assume), I don't think we want / need "noopener noreferrer".  It doesn't hurt much but might mess with our analytics a tiny bit if we lose referrer info for our own internal links. 🤷

Reply:
oh true!